_episodes/02-setup.md

```diff
@@ -143,19 +143,219 @@ $ git config --list
+
+
+~~~
+The authenticity of host 'github.com (192.30.255.112)' can't be established.
+RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
+This key is not known by any other names
+Are you sure you want to continue connecting (yes/no/[fingerprint])? y
+Please type 'yes', 'no' or the fingerprint: yes
+Warning: Permanently added 'github.com' (RSA) to the list of known hosts.
+git@github.com: Permission denied (publickey).
+~~~
+{: .output}
+
+Right, we forgot that we need to give GitHub our public key!  
+
+First, we need to copy the public key.  Be sure to include the `.pub` at the end, otherwise you’re looking at the private key. 
+
+~~~
+cat ~/.ssh/id_ed25519.pub
+~~~
+{: .language-bash}
+
+~~~
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDmRA3d51X0uu9wXek559gfn6UFNF69yZjChyBIU2qKI vlad@tran.sylvan.ia
+~~~
+{: .output}
+
+> ## Copy and paste in the unix shell
+> Have you tried to use your usual keyboard hotkeys for Copy and Paste in the Unix shell and found they didn't work? 
+> 
+> If you're using GitBash in Windows, you can copy and paste using your mouse.
+> Highlight the text you want to copy, then right click and select `copy`.
+> Similarly for paste, right click at the prompt and select `paste`. 
+> 
+> On Windows in GitBash, you can use `ctrl + INSERT` to copy and `Shift + INSERT` to paste.
+> 
+> For many users on a mac, the `cmd + c` and `cmd + v` work as expected in Terminal.
+> 
+> Linux terminals will vary by distro, but try `Shift + ctrl + c` and `Shift + ctrl + v` in Bash.
+{: .callout}
+
+Now, going to GitHub.com, click on your profile icon in the top right corner to get the drop-down menu.  Click "Settings," then on the 
+settings page, click "SSH and GPG keys," on the left side "Account settings" menu.  Click the "New SSH key" button on the right side. Now, 
+you can add the title (Dracula uses the title "Vlad's Lab Laptop" so he can remember where the original key pair
+files are located), paste your SSH key into the field, and click the "Add SSH key" to complete the setup.
+
+Now that we’ve set that up, let’s check our authentication again from the command line. 
+~~~
+$ ssh -T git@github.com
+~~~
+{: .language-bash}
+
+~~~
+Hi Vlad! You've successfully authenticated, but GitHub does not provide shell access.
+~~~
+{: .output}
+
+Good! This output confirms that the SSH key works as intended. We are now ready to push our work to the remote repository.
+
+Now that authentication is setup, we can return to the remote.  This command will push the changes from
+our local repository to the repository on GitHub:
+
+~~~
+$ git push origin main
+~~~
+{: .bash}
+
+~~~
+Counting objects: 9, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (6/6), done.
+Writing objects: 100% (9/9), 821 bytes, done.
+Total 9 (delta 2), reused 0 (delta 0)
+To https://github.com/vlad/planets
+ * [new branch]      main -> main
+Branch main set up to track remote branch main from origin.
+~~~
+{: .output}
+
```

Comment:
Don't need this section in the setup episode and it won't work since we don't have a repo yet.

Reply:
maybe the proxy stuff too? not sure about that as I've never had to use it

Reply:
I took out the git push part, but I'm thinking we should leave the proxy part in. It seems out of scope for this PR. I'll create an issue so we don't forget about it.